Source/WebCore/platform/graphics/cpu/arm/filters/FEBlendNeonApplier.cpp

```diff
@@ -176,19 +176,19 @@ bool FEBlendNeonApplier::apply(const Filter&, const FilterImageVector& inputs, F
     if (!destinationPixelBuffer)
         return false;
 
-    auto* destinationPixelArray = destinationPixelBuffer->bytes();
+    auto* destinationPixelArray = destinationPixelBuffer->bytes().data();
```

Comment:
This is a use-after-free because the std::span is returned by value and is a temporary.

You'll need to fix this throughout this pull request.

Reply:
That’s incorrect. The `span` is just a struct containing a pointer and length. Its lifetime is unimportant. What’s important is the lifetime of the bytes in the pixel buffer and this has not changed.

This code is fine.

Reply:
I see I got very confused. You're right; `span` is non-owning. And that's why we copy it by value. OK, I'll need to get used to this....